website/src/site/apt/team.apt

```diff
@@ -69,7 +69,7 @@ Team
 *------------*--------------------------*---------------------------*
   jerryjung |       You Sun Jung       |         SK Telecom        |
 *------------*--------------------------*---------------------------*
-  dongjoon  |      Dongjoon Hyun       |                           |
+  dongjoon  |      Dongjoon Hyun       |         HortonWorks       |
```

Comment:
We've voted some time ago to allow doing minor changes to non-code files (documentation) without a JIRA and PR. This change would have been a good candidate for such approach.